_rules/document-title-is-descriptive-c4a8a4.md

```diff
@@ -178,7 +178,7 @@ Even though a correct `<title>` element is put in the `<head>` of the document,
 
 #### Inapplicable Example 1
 
-This document has a `<title>` element but is inapplicable since the document element is an SVG `<svg>` element.
+This document has a `<title>` element but is inapplicable since the document element is an `SVG` element.
```

Comment:
I think this should be "is an SVG `svg` element"
=> it's an SVG element, not an HTML element.
=> the element name is `svg`, not `circle` or `title`.